roles/kube_prometheus/files/grafana_ingress.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "kube-prometheus-grafana-ingress"
```

Comment:
Could make sense to add labels here as well over time, cfr https://github.com/scality/metal-k8s/pull/30#discussion_r176879125

```yaml
app: kube-prometheus-grafana
release: kube-prometheus
```